### DISCUSSION TITLE
How to do highlight on yank？

### DISCUSSION THREAD

**Discussion by Old-Farmer** [vscode-neovim/vscode-neovim#2176] [Q&A] [answered]
```lua
  vim.api.nvim_create_autocmd("TextYankPost", {
    callback = function()
      vim.highlight.on_yank()
    end,
  })
```

How to configure to achieve the same effect using the above autocmd in neovim? If this is duplicated or cannot be achieved, please let me know.

**Comment by xiyaowong** [comment, marked as answer]
Try `vim.highlight.on_yank({ higroup = 'Search' })`
